api/src/main/java/org/openmrs/module/fhir/api/strategies/relatedperson/RelatedPersonStrategy.java

```diff
@@ -27,6 +29,31 @@ public RelatedPerson getRelatedPerson(String uuid) {
 		return FHIRRelatedPersonUtil.generateRelationshipObject(omrsRelationship);
 	}
 
+	/**
+	 * @see org.openmrs.module.fhir.api.strategies.relatedperson.GenericRelatedPersonStrategy#searchByPatientIdentifier(String)
+	 */
+	@Override
+	public List<RelatedPerson> searchByPatientIdentifier(String identifierValue) {
+		List<RelatedPerson> relatedPersonList = new ArrayList<>();
+		List<Relationship> relationships = new ArrayList<>();
+		List<PatientIdentifierType> allPatientIdentifierTypes = Context.getPatientService().getAllPatientIdentifierTypes();
```

Comment:
What is the reason for loading all patient identifier types from the database instead of just passing null to Context.getPatientService().getPatients() for the identifier types parameter?